mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -629,9 +626,13 @@ public void run() {
                     int interestOps = channel.getInternalInterestOps();
```

Comment:
The commit you gave did not affect that file. I found the change in commit https://github.com/kaazing/gateway/commit/105d1c6c39419c95840b956522c035c2e0c10652

If I'm adding this piece of code in register and deregister, it means that I can remove all the changes that I did ? I'll try to check it.